pennylane/transforms/__init__.py

```diff
@@ -20,38 +20,32 @@
 Transforms
 ----------
 
-Transforms that act on QNodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+How to build a custom PennyLane transform?
```

Comment:
kind of a nitpick but we don't usually have questions in doc headers
```suggestion
How to build a custom PennyLane transform
```

Reply:
The heading underline will also need to be adjusted :)

Reply:
Thanks